cmd/frr-exporter/main.go

```diff
@@ -63,6 +63,6 @@ func main() {
 	err = server.ListenAndServe()
 	// Run server
 	if err != nil {
-		log.Fatal(fmt.Errorf("Can't start server %w", err))
+		log.Fatal(fmt.Errorf("failed to start server %w", err))
```

Comment:
usually you would put a colon before the wrapped error
```suggestion
		log.Fatal(fmt.Errorf("failed to start server: %w", err))
```